lib/datadog/appsec/contrib/rack/request_middleware.rb

```diff
@@ -150,7 +150,8 @@ def add_appsec_tags(processor, scope)
 
             return unless trace && span
 
-            span.set_tag('_dd.appsec.enabled', 1)
+            span.set_tag(Datadog::AppSec::Ext::TAG_APPSEC_ENABLED, 1)
+            span.set_tag(Datadog::AppSec::Ext::TAG_APM_ENABLED, 0) if Datadog.configuration.appsec.standalone.enabled
```

Comment:
not sure, but does this need a comment?

Reply:
I would add somethink like "we want to make sure the customer will not get billed for APM when standalone ASM is enabled". But again, it's probably fine to leave it as it is

Reply:
I read PR description but did not understand that this is what the PR is trying to achieve. So probably would be helpful to include this bit *somewhere*.

Reply:
I added a comment, thanks!